autoload/neomake/makers/ft/go.vim

```diff
@@ -47,3 +47,19 @@ function! neomake#makers#ft#go#govet()
             \ '%-G%.%#'
         \ }
 endfunction
+
+let g:neomake_go_gometalinter_args =
+    \ get(g:, 'neomake_go_gometalinter_args', ['--disable-all',
+        \ '--enable=varcheck', '--enable=structcheck', '--enable=ineffassign',
+        \ '--enable=unconvert', '--enable=gosimple', '--enable=staticcheck'])
+
+function! neomake#makers#ft#go#gometalinter()
+    return {
+        \ 'exe': 'gometalinter',
+        \ 'args': g:neomake_go_gometalinter_args,
+        \ 'append_file': 0,
+        \ 'cwd': '%:h',
+        \ 'mapexpr': 'neomake_bufdir . "/" . v:val',
+        \ 'errorformat': '%W%f:%l:%c:%m',
```

Comment:
I would say yes, gometalinter simply copies the output from the various tools and all those tools have their own probability of false positives - if it compiles, then I think everything else is a warning.